orchagent/fdborch.cpp

```diff
@@ -159,7 +159,8 @@ void FdbOrch::update(sai_fdb_event_t type, const sai_fdb_entry_t* entry, sai_obj
         break;
 
     case SAI_FDB_EVENT_FLUSHED:
-        if (bridge_port_id == SAI_NULL_OBJECT_ID && entry->bv_id == SAI_NULL_OBJECT_ID)
+        if ((bridge_port_id == SAI_NULL_OBJECT_ID && entry->bv_id == SAI_NULL_OBJECT_ID)
+            || (bridge_port_id && entry->bv_id == SAI_NULL_OBJECT_ID))
```

Comment:
BTW, I just cherry-picked these changes, I think we need more clean up here. Because the else if is doing the same and thing and this particular if can be reduced to
```
if (entry->bv_id == SAI_NULL_OBJECT_ID)
```